base_export_async/data/config_parameter.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="attachment_time_to_live" model="ir.config_parameter">
+        <field name="key">attachment.time.to.live</field>
```

Comment:
I'd rename this as `base_export_async.attachment_TTL`

Reply:
Usually we don't write xml_id with Uppercase so I will not do that.

Reply:
keeping lower case, doesn't matter. as long as it contains the module name ;)